### PR TITLE
Unit tests for MusicAlbum and Genre classes

### PR DESCRIPTION
**Pull request by abdozayan12**
Add unit tests for all implemented methods.

### CODE REVIEW

File: src/spec/genre_spec.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    @genre.add_item('Rock')
+    expect(@genre.items).to include('Rock')
+  end
+
+  it 'the length of items is equal to the number of items added' do
+    @genre.add_item('Rock')
+    @genre.add_item('Pop')
+    expect(@genre.items.length).to eq(2)
+  end
+end
```
Comment (marvin-nyalik) [inline src/spec/genre_spec.rb:39]: Hi @abdozayan12, Great job with the unit tests. I however noticed that the above 2 tests are failing. Kindly make the necessary changes to allow all tests to pass.

File: src/spec/music_album_spec.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  end
+
+  it 'is an instance of MusicAlbum' do
+    expect(@music_album).to be_an_instance_of(MusicAlbum)
+  end
+
+  it 'has an id number' do
+    expect(@music_album.id).to be_an(Integer)
+  end
+
```
Comment (marvin-nyalik) [inline src/spec/music_album_spec.rb:16]: For some reason, all the tests for the MusicAlbum are failing. Kindly inspect the code and ensure the results are positive.
music_test

**Review by marvin-nyalik — CHANGES_REQUESTED** [review]
Hi @abdozayan12, you made great progress with the tests.

In this unit testing branch, some tests are passing while others are not. Please review the code and make appropriate changes for all tests to be positive. Cheers 🎉.

**Review by migo1 — APPROVED** [review]
Hi @abdozayan12  

Good job!

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Tests are passing

### Approved 👏👏👏

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Pull request merged**